pom.xml

```diff
@@ -208,6 +210,14 @@
           <suppressionsLocation>${basedir}/src/main/resources/checkstyle/checkstyle-suppressions.xml</suppressionsLocation>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs.plugin.version}</version>
+        <configuration>
+          <excludeFilterFile>${basedir}/sb-excludes.xml</excludeFilterFile>

```

Comment:
Can you change this to `${basedir}/src/main/resources/spotbugs/spotbugs-exclude-filter.xml` and move the exclude file there. 

I will separately move checkstyle.xml to the src folder too. No point to clutter up the root directory with things.

Reply:
done